vividus-plugin-web-app/src/main/java/org/vividus/selenium/logging/BrowserLogManager.java

```diff
@@ -59,13 +59,14 @@ public static void resetBuffer(WebDriver driver, boolean ignoreUnsupportedDriver
     private static Optional<LogEntries> getLog(WebDriver driver, boolean ignoreUnsupportedDrivers)
     {
         // The Selenium log API isn't supported: https://github.com/w3c/webdriver/issues/406
-        if (WebDriverManager.isBrowserAnyOf(driver, Browser.FIREFOX, Browser.IE))
+        // Safari: https://developer.apple.com/documentation/webkit/macos_webdriver_commands_for_safari_11_1_and_earlier
+        if (WebDriverManager.isBrowserAnyOf(driver, Browser.FIREFOX, Browser.IE, Browser.SAFARI))
         {
             if (ignoreUnsupportedDrivers)
             {
                 return Optional.empty();
             }
-            throw new IllegalStateException("Firefox does not support retrieval of browser logs");
+            throw new IllegalStateException("Browser does not support retrieval of browser logs");
```

Comment:
why not to preserve browser name?

Reply:
this statement is applicable for one member team, if we have a huge QA team and someone is assigned to triage regression suite report it might be useful taking into account the fact that we can specify browser name different than specified in properties/profiles

Reply:
Well, that is true, but the error is pretty clear. Once you get it - there are ways to find the browser used for the story.
Previous implementation always reported firefox which is really strange behaviour :)

 @valfirst  could you please add you opinion